apps/src/labs/projects/SourcesStore.ts

```diff
@@ -35,7 +35,9 @@ export class RemoteSourcesStore implements SourcesStore {
   private lastSaveTime: number | null = null;
 
   async load(channelId: string) {
+    console.groupCollapsed('fetching main.json');
```

Comment:
Unfortunately, yes, that would be the case - it's either always the `fetching main.json` string, or the 404 error when it doesn't exist.

So with that, I'll close out this PR and the ticket, since I don't think there's a better way to resolve it.